packages/edit-navigation/src/components/layout/style.scss

```diff
@@ -7,3 +7,15 @@
 		padding-left: 0;
 	}
 }
+
+.edit-navigation-layout {
+	display: flex;
+	flex-direction: column;
+	// Make the navigation layout full-height, accounting for wp-admin top bar and other furniture.
+	height: calc(100vh - 146px);
+
+	// Wp admin top bar becomes less-chunky, so a smaller subtraction is needed.
+	@include break-medium() {
+		height: calc(100vh - 97px);
```

Comment:
This line seems to be the cause of some weirdness with the vertical scrollbar flickering in and out of existence when hovering over the list view or opening dropdowns.

Reply:
@tellthemachines Thanks for testing, would you be able to share repro steps and maybe screenshots? I didn't spot anything locally apart from some weirdness with the List View when it's big:
<img width="395" alt="Screenshot 2021-01-28 at 4 15 16 pm" src="https://user-images.githubusercontent.com/677833/106109129-1f155080-6184-11eb-8abb-efd89adb828e.png">

Reply:
Yup, so it's happening on Chrome/macOS, here's a gif:

![ezgif com-gif-maker (1)](https://user-images.githubusercontent.com/8096000/106217216-e11e3800-6228-11eb-98dc-47be07f0fc07.gif)

It might be related to the horizontal scrollbar, as that's only happening on Chrome too (though that's not specific to your branch). 

In testing across environments, I also realised that there can be a notice under the admin bar, and that'll always cause a vertical scrollbar to appear, because the `100vh - x` isn't taking it into account:

![notice-admin](https://user-images.githubusercontent.com/8096000/106217482-715c7d00-6229-11eb-9423-3d0552b89b87.png)

My suggestion is, instead of using `vh`, let's leverage the page-specific `gutenberg_page_gutenberg-navigation` body class and try something like:

```
.gutenberg_page_gutenberg-navigation {
    #wpwrap, #wpcontent, #wpbody, #wpbody-content, .edit-navigation, .components-drop-zone__provider, .edit- 
    navigation-layout {
        display: flex;
        flex-direction: column;
        flex-grow: 1;
    }
}
```

Reply:
That's a lot better, thanks for saving me from CSS hell 😆